examples/advanced/flutter/style_dictionary/.metadata

```diff
@@ -0,0 +1,10 @@
+# This file tracks properties of this Flutter project.
+# Used by Flutter tool to assess capabilities and perform upgrades etc.
+#
+# This file should be version controlled and should not be manually edited.
+
+version:
+  revision: 1946fc4da0f80c522d7e3ae7d4f7309908ed86f2
+  channel: dev
+
+project_type: package
```

Comment:
I don't know flutter well; should this file be included or added to the .gitignore?

Reply:
It should be included